english/wake-up/wake-up-again/dream2/dream2.md

```diff
@@ -0,0 +1,3 @@
+You look up and see beautiful blue sky. It is a bright sunny day. 
+You look around and you see big white cyclinders everywhere on all four sides. 
+Suddenly an aircraft flies by. In the shiny underbelly of the airplane you see a sea of white cyclinders stretched across the landscape and realize that you are just a marshmallow among thousands in a vast island of marshmallows. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.